security/mk-cacerts.sh

```diff
@@ -105,11 +103,26 @@ for FILE in certs/*.crt; do
             -file "$FILE" \
             -keystore "cacerts" \
             -storepass "changeit"
-
-            IMPORTED+=("${ALIAS}")
+        else
+            # Importing using OpenJDK GenerateCacerts, so must ensure alias is a valid filename
+            ALIAS_NO_INVALID="${ALIAS//[ :()]/_}"
+            ALIAS_FILENAME=$(echo "${ALIAS_NO_INVALID}" | tr -cd '0-9a-zA-Z,_' | tr '[:upper:]' '[:lower:]')
+            echo "Renaming $FILE to certs/$ALIAS_FILENAME"
+            if test -f "certs/$ALIAS_FILENAME"; then
+                echo "ERROR: Certificate alias file already exists certs/$ALIAS_FILENAME"
+                echo "security/mk-cacerts.sh needs ALIAS_FILENAME filter updating to make unique"
+                exit 1
+            fi
+            mv "$FILE" "certs/$ALIAS_FILENAME"
         fi
-    else
-         echo "Renaming $FILE to certs/$ALIAS"
-         mv "$FILE" "certs/$ALIAS"
+
+        IMPORTED+=("temurin_${ALIAS}_temurin")
```

Comment:
guess to add termurin_ _temurin does not really make a difference? but for visibility

Reply:
It does, as it prevents the already imported logic missing Certs:
```
    if printf '%s\n' "${IMPORTED[@]}" | grep "temurin_${ALIAS}_temurin"; then
```
As the current logic skips "QuoVadis Root CA 3" as it is a substring of "QuoVadis Root CA 3 G3"
See current build logs:
```
13:06:45  C=BM,O=QuoVadis Limited,CN=QuoVadis Root CA 3 G3
13:06:45  Skipping certificate with alias: C=BM,O=QuoVadis Limited,CN=QuoVadis Root CA 3 as it already exists
```

Reply:
right! so if the "subject=C = BM, O = QuoVadis Limited, CN = QuoVadis Root CA 2 G3"  has been added before "subject=C = BM, O = QuoVadis Limited, CN = QuoVadis Root CA" it does make a difference